doc/yaml.md

```diff
@@ -380,6 +379,7 @@ YAML block sequences are also disallowed in code mode.
 The rationale here is that if you ever see `{`, `[`, `- ` or `>' (folded scalar)
 you can assume that you are in data mode and not code mode.
 
+
```

Comment:
There seems to be a contradiction:
On the one hand:
"Since [] vectors and {} maps are so common in YAMLScript expressions.."
On the other hand
"The rationale here is that if you ever see {, [,  you can assume that you are in data mode"

Reply:
Yeah I meant common in Clojure but was trying to avoid talking about Clojure here.
I'll try again.